android/app/src/main/res/values-ca/strings.xml

```diff
@@ -733,6 +733,22 @@
 	<string name="osm_wiki_about_url">https://wiki.openstreetmap.org/wiki/Ca:About</string>
 	<!-- Tip #00 -->
 	<string name="app_tip_00">Gràcies per utilitzar els nostres mapes creats per la comunitat!</string>
+	<!-- Text on the Android Auto placeholder screen that maps are displayed on the phone screen -->
+	<string name="aa_used_on_the_phone_screen">Ara esteu utilitzant els Organic Maps a la pantalla del telèfon</string>
+	<!-- Text on the phone placeholder screen that maps are displayed on the car screen -->
+	<string name="aa_used_on_the_car_screen">Ara esteu utilitzant els Organic Maps a la pantalla del cotxe</string>
+	<!-- Displayed on the phone screen. Android Auto connected -->
+	<string name="aa_connected_title">Esteu connectat a Android Auto</string>
+	<!-- Displayed on the phone screen. Button to display maps on the phone screen instead of a car -->
+	<string name="aa_continue_on_the_phone">Continueu al telèfon</string>
+	<!-- Displayed on the Android Auto screen. Button to display maps on the car screen instead of a phone. Must be no more than 18 symbols! -->
+	<string name="aa_continue_in_the_car">A la pantalla del cotxe</string>
+	<!-- Ask user to grant location permissions -->
+	<string name="aa_location_permissions_request">Aquesta aplicació requereix accés a la vostra ubicació per a la navegació.</string>
+	<!-- Displayed on the Android Auto screen. Grant Permissions button. Must be no more than 18 symbols! -->
+	<string name="aa_grant_permissions">Concedir permisos</string>
```

Comment:
"Concedeix permisos". In Catalan localization guidelines by Softcatalà, commands use the imperative, not the infinitive, as Spanish does. Though I know that some phone OSes use the infinitive nevertheless, like iOS and EMUI.